cdk-deployment/lambda/Dockerfile

```diff
@@ -6,10 +6,10 @@ COPY . /code/
 
 RUN mkdir -p /asset && \
 	python -m pip install --upgrade pip && \
-	pip install mangum uvicorn fastapi[all] -t /asset
+	pip install stac-validator mangum uvicorn fastapi[all] -t /asset --no-binary pydantic
 
-RUN pip install /code -t /asset
+# RUN pip install -e /code -t /asset
+RUN ls
+RUN cp /code/lambda.py /asset/
 
-RUN cp /code/cdk-deployment/lambda/lambda.py /asset/lambda.py
-
-CMD ["echo", "hello world"]
\ No newline at end of file
+CMD ["lambda.handler"]
```

Comment:
This doesn't get used. The dockerfile is only used for bundling. Once the zip is created, it is pushed to S3 and ran as a standard lambda function.